src/components/visual-description/visual-description.tsx

```diff
@@ -2,7 +2,17 @@ import React from 'react';
 import cx from 'classnames';
 import styles from './visual-description.module.scss';
 
-const VisualDescription = ({
+interface VisualDescriptionProps {
+  title: string;
+  ariaLabel: string;
+  children: React.ReactNode;
+  description: React.ReactNode;
```

Comment:
I see from the usages in `monday-ui-react-core` that it's a string
```suggestion
  description: string;
```